artemis-server/src/main/java/org/apache/activemq/artemis/core/replication/ReplicationEndpoint.java

```diff
@@ -203,11 +203,9 @@ public void handlePacket(final Packet packet) {
             ActiveMQServerLogger.LOGGER.invalidPacketForReplication(packet);
          }
       } catch (ActiveMQException e) {
-         logger.warn(e.getMessage(), e);
```

Comment:
why take it off? 

I was bitten the other day at a customer site for not having enough information on what's happening... please keep it

Reply:
got it